### ISSUE TITLE
v1.9.4:: My ticket column :: answered ticket status

### ISSUE THREAD

**Issue by henrycoule** [osTicket/osTicket#1519]
I am not too sure how to even word the title and also whether this is a bug or question.  Before 1.9.4 we usually mark tickets as answered so it moves to "My Tickets" queue until we have resolved the issue and change the status to close. Now with the new ticket statuses, there is no way to mark a ticket as answered without actually answering it. Additionally, all the other ticket statuses (custom define, etc) are all based on either closed or open, so even if I create a custom status like "In Progress" it will either be closed or open. We don't want to have a ticket which is currently waiting on a user or something to be done to be in the open queue neither do we want it to be closed but marked as answered and placed in the "My Ticket" queue.

How do we make this happen? I noticed one of our users has the ability to mark a ticket as answered meanwhile we all have the same permissions and we do not see any such option when we click on "More" when viewing a ticket.

![sc_more](https://cloud.githubusercontent.com/assets/6393255/5172355/cc31d520-73e4-11e4-9c7d-00c0ac457bec.png)
